cypress/features/regression/sidebar.feature

```diff
@@ -5,6 +5,12 @@ Feature: Sidebar component
     Given I open "Sidebar" component page
```

Comment:
also, we need to add a separate test for golden border

Reply:
added: 9af655a